app/google-services.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
There is two clients:
 - one is for production version
 - second for development
 How about qa? Have you checked production version?